### PR TITLE
PAYEE-262: Upgrade the FE Library for Check previous tax years

### PR DESCRIPTION
**Pull request by vivekhmrc**
Signed-off-by: vivekhmrc <vivek.kumar@digital.hmrc.gov.uk>

- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Comment by sergi-hmrc** [comment]
I've noticed the prototype shows `You can also [check how much Income Tax you paid ](http://localhost:9416/tax-you-paid/status)for 6 April 2021 to 5 April 2022.` for all views but we only show that for the previous year. I'm not sure about which is the expected behaviour:

File: app/views/TaxCodeDetailsPreviousYearsView.scala.html [hunk 1]
```diff
@@ -20,45 +20,54 @@
...
 }
 
-@mainTemplate(
+@main(
     title = taxCodeViewModel.title,
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = Some("full-width")){
+    backLinkContent=messages("tai.back-link.upper"),
+    backLinkUrl = Some("javascript:history.go(-1)"),
```
Comment (sergi-hmrc) [inline app/views/TaxCodeDetailsPreviousYearsView.scala.html:41]: should we use javascript or `Some(routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage.url),
`?

Comment (vivekhmrc) [reply app/views/TaxCodeDetailsPreviousYearsView.scala.html]: 👍  Incorporated the change

**Review by sergi-hmrc — APPROVED** [review]

File: app/views/TaxCodeDetailsPreviousYearsView.scala.html (hunk 1 above)
Comment (brendanstraughan) [reply app/views/TaxCodeDetailsPreviousYearsView.scala.html]: is it worth checking with ux to see if this is the page we want to be sending the user back to?

File: app/views/MainTempleteNew.scala.html [hunk 2]
```diff
@@ -158,11 +159,11 @@
     }
 
     @attorneyBanner
-
     @backLinkUrl.map(url =>
-        govukBackLink(BackLink(attributes = Map("id" -> "returnToChooseTaxYearLink"), href = url, content = Text(backLinkContent)))
+        govukBackLink(BackLink(classes = "js-visible-back",
```
Comment (brendanstraughan) [inline app/views/MainTempleteNew.scala.html:163]: Not sure if it makes a difference but I done this with Mas and he said to use "js-visible"?

Comment (vivekhmrc) [reply app/views/MainTempleteNew.scala.html:163]: I rechecked and in the PROD the same is being used. Will recheck with Mas on return though.

File: app/views/TaxCodeDetailsPreviousYearsView.scala.html (hunk 1 above)
Comment (vivekhmrc) [reply app/views/TaxCodeDetailsPreviousYearsView.scala.html]: Verified 👍

**Pull request merged**